pkgs/servers/home-assistant/default.nix

```diff
@@ -252,6 +252,7 @@ in with py.pkgs; buildPythonApplication rec {
     pyyaml
     requests
     ruamel_yaml
+    setuptools
```

Comment:
Given that the comment above these packages says 

>    # Only packages required in setup.py

i think we should add move this out of this list, maybe above it, and add a separate comment.

Reply:
Or maybe we should make an upstream PR adding it?